mlx-macros/src/lib.rs

```diff
@@ -51,3 +51,51 @@ pub fn default_device(_attr: TokenStream, item: TokenStream) -> TokenStream {
 
     TokenStream::from(expanded)
 }
+
+#[proc_macro_derive(GenerateDTypeTestCases)]
```

Comment:
Minor thing, but I think it should be `GenerateDtypeTestCases` if we were to follow the rust naming convention strictly